test/json_schema/reference_expander_test.rb

```diff
@@ -173,6 +174,12 @@
     # but diving deeper results in the same pointer again
     schema = schema.properties["app"]
     assert_equal "#/properties/app", schema.pointer
+
+    schema = @schema.properties["app1"]
+    assert_equal "#/properties/app1", schema.pointer
+
+    schema = schema.properties["app1"]
+    assert_equal "#/properties/app1", schema.pointer
```

Comment:
I didn't understand what this test did, and I copied https://github.com/okitan/json_schema/blob/fix_50/test/json_schema/reference_expander_test.rb#L170-L176 .

Do you think it is better to remove?